js/dialogcards.js

```diff
@@ -490,9 +492,14 @@ H5P.Dialogcards = (function ($, Audio, JoubelUI) {
     // Check if card has been turned before
     var turned = $c.hasClass('h5p-dialogcards-turned');
 
+    //Show the front image only if a back image is not uploaded.
+    //P.S. Adding this snippet after changeText() caused image sizing problems
+    if (self.params.dialogs[$card.index()].backImage.length) {
```

Comment:
Also backImage is not an array, it is an object, so we should probably not check its length, I suppose checking that it is defined would suffice.